resources/views/email/signup/user.blade.php

```diff
@@ -4,11 +4,20 @@
         Hello **{{ $user["first_name"] }} {{ $user["last_name"] }},**
 
         You have successfully been registered on the SpringVerse platform.
-        Please find your login credentials below:
 
-        * Email - {{ $user['email'] }}
-        * Default Password - {{ $defaultPassword }}
+        @if($registration_source == "BACKEND")
+            {
+            Please find your login credentials below:
 
+            * Email - {{ $user['email'] }}
+            * Default Password - {{ $defaultPassword }}
+            }
+        @else{
+
+        }
+        @endif
+
+        <a href="http://www.springverse.com/login">Login</a>
```

Comment:
You need to use the APP_URL environment variable here, and not hardcode a url

Reply:
Resolved the changes